deployment/overlays/dev/kustomization.yaml

```diff
@@ -15,12 +15,12 @@ patches:
   patch: |-
     - op: replace
       path: '/spec/template/spec/containers/0/imagePullPolicy'
-      value: Never
+      value: Always
```

Comment:
Not surprisingly, I think we've come up with different local development/testing workflows. It'd probably be good to put our heads together and see if we can come up with a process that works for both Windows/WSL and Mac systems.

The reason for the `imagePullPolicy: Never` here was that I was building images but _not_ pushing them to a remote registry. I was then loading them into a `kind` cluster so that they were available to the nodes. If the container runtime ever tried to pull the images it would obviously fail, because they didn't exist remotely.

While this workflow felt convenient to me, I've never worked on a Mac, and so there may be reasons I failed to think about why this wouldn't work for other people.

Reply:
It makes sense now and thanks for the clarification. the file change slipped into the commit accidentally.

my workflow consists of running local test on an external cluster, not locally hosted like kind or minikube. with `Never` it would not pick up updated images. 

i think a better approach for the [policy](https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy) might be to use `IfNotPresent` so that
- behaves like Never if using `kind` and image updated / exist through other channels,
- pull if not
- dev could update image tag (recommended) to force a new image to be pull from remote repo.